aexpect/utils/genio.py

```diff
@@ -9,18 +9,29 @@
 #
 # See LICENSE for more details.
 
+"""
+Naive module that keeps tacks of some opened files and somehow manages them.
+"""
```

Comment:
Yep, I agree that this should go eventually. But first things first

Reply:
Yep, I wanted to turn it into a public interface or remove, but when I saw how is it used in Avocado-vt I just created: https://trello.com/c/6zsHUXYr/1351-aexpect-get-rid-of-aexpectutilsgenio